metadata/base/metadata-envoy-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: metadata-envoy
```

Comment:
please name it `envoy-deployment`, string prefix `metadata-` is automatically added.